### PR TITLE
Clarify maintainance rules

### PR DESCRIPTION
**Pull request by niccokunzmann**
As with #26, this should clarify who is to be contacted and how to become a maintainer.

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

- [X] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:

This should clarify how to become a maintainer and what it means to be one.

#### Changes proposed in this pull request:

- maintainance rules

@jajodiaraghav, please review and merge
@S2606, please comment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,8 +74,24 @@ them together into one.
...
+2. You or someone else proposes you in an issue to become a member of the team.
+3. A Badgeyay admin adds you.
+
+To be excluded from the team:
+1. Do nothing or violate the rules of [CCCC][cccc] or [Badgeyay](.github/CONTRIBUTING.md) willingly in a harmful way. 
+2. Someone creates an issue to ask for removal.
+3. An admin removes you.
+
+
+[gitter]: https://gitter.im/fossasia/open-event-orga-server
```
Comment (S2606) [inline README.md]: @niccokunzmann change it to https://gitter.im/fossasia/badgeyay and then its good for merge

Comment (jajodiaraghav) [reply README.md]: We do not have a chat room for badgeyay yet! We are currently using open event room in gitter.

File: README.md [hunk 2]
```diff
@@ -74,8 +74,24 @@ them together into one.
 Maintainers
 -------------------
 
-The project is maintained by
- - Nicco Kunzmann ([@niccokunzmann](https://github.com/niccokunzmann))
- - Tarun Kumar ([@meets2tarun](https://github.com/meets2tarun))
- 
- [gitter]: https://gitter.im/fossasia/open-event-orga-server
+You can reach the maintainers,
+ping them personally by loooking at the [Badgeyay team][team].
```
Comment (jajodiaraghav) [inline README.md]: You spelled "looking" wrong

File: README.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ If you like to join developing,
 - you can find/create [issues](https://github.com/fossasia/badgeyay/issues) and solve them.
   - When you solve an issue, you do not own it. Share your progress via a Pull-Requst as soon as possible.
   - Discuss with others who work on the issue about the best solution. It is your responsibility, not the maintainer's to choose the best solution.
-- If in doubt, let's follow [CCCC](https://rfc.zeromq.org/spec:42/C4).
+- If in doubt, let's follow [CCCC][cccc].
```
Comment (jajodiaraghav) [inline README.md]: Can we remove this line?

File: README.md (hunk 1 above)
Comment (S2606) [reply README.md]: @jajodiaraghav yes we have chat room. Check out this link below

 https://gitter.im/fossasia/badgeyay

File: README.md [hunk 4]
```diff
@@ -74,8 +74,24 @@ them together into one.
...
+Please see your role in the [CCCC][cccc].
+
+The project is maintained by the [Badgeyay maintainer's team][team].
+To join the team:
+1. Contribute
+2. You or someone else proposes you in an issue to become a member of the team.
+3. A Badgeyay admin adds you.
+
+To be excluded from the team:
+1. Do nothing or violate the rules of [CCCC][cccc] or [Badgeyay](.github/CONTRIBUTING.md) willingly in a harmful way.
```
Comment (S2606) [inline README.md]: Instead of `do nothing..` start with `do not violate`

File: README.md (hunk 1 above)
Comment (jajodiaraghav) [reply README.md]: Sorry my bad!

**Comment by niccokunzmann** [comment]
@S2606 @jajodiaraghav What you you think?

**Comment by niccokunzmann** [comment]
I addressed the changes.

File: README.md [hunk 5]
```diff
@@ -74,8 +74,24 @@ them together into one.
...
+Please see your role in the [CCCC][cccc].
+
+The project is maintained by the [Badgeyay maintainer's team][team].
+To join the team:
+1. Contribute
+2. You or someone else proposes you in an issue to become a member of the team.
+3. A Badgeyay admin adds you.
+
+To be excluded from the team:
+1. Follow the rules of [CCCC][cccc] or [Badgeyay](.github/CONTRIBUTING.md) and do not violate them willingly or in a harmful way.
```
Comment (S2606) [inline README.md]: you can represent the first point as
`Do not violate the rules of .... or .... in  a harmful way`

**Comment by jajodiaraghav** [comment]
Thanks! This looks good.
Merging

File: README.md (hunk 5 above)
Comment (niccokunzmann) [reply README.md]: oh...

**Pull request merged**